examples/src/main/java/com/teamdev/jxbrowser/examples/SaveWebPage.java

```diff
@@ -61,7 +61,8 @@ public void windowClosing(WindowEvent e) {
             frame.setVisible(true);
         });
 
-        browser.navigation().loadUrlAndWait("https://www.google.com");
+        browser.navigation()
+               .loadUrlAndWait("https://webglsamples.org/dynamic-cubemap/dynamic-cubemap.html");
```

Comment:
It's not a good idea to save a web page with WebGL content. It should be the html5test page.